src/js/components/TextInput/doc.js

```diff
@@ -199,7 +199,7 @@ export const themeDoc = {
   'textInput.disabled.opacity': {
     description: 'The opacity when the textInput is disabled.',
     type: 'number',
-    defaultValue: 0.3,
+    defaultValue: 'global.control.disabled.opacity',
```

Comment:
Good catch, thanks for addressing it.
According to base.js, the defaultValue should be `undefined`
```
textInput: {
      // extend: undefined,
      // disabled: { opacity: undefined },
    },
```